systemtests/tests/testfind/etc/bareos/bareos-fd.d/client/myself.conf.in

```diff
@@ -0,0 +1,19 @@
+Client {
```

Comment:
is the bareos-fd.d directory required?

Reply:
it is not required, knew this after testing